src/server/pps/server/transaction_defer.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/transactionenv/txncontext"
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+)
+
+// Propagater is an object that is used to propagate PFS branches at the end of
```

Comment:
update comment (maybe struct name, too - JobStarter or something?)

Reply:
The action has been renamed to `PropagateJobs` - `JobStarter` doesn't really work because it just creates the jobs rather than starting them (it's the worker that ends up doing that), so maybe `JobCreator` could work...